libvcx/src/aries/handlers/proof_presentation/prover/state_machine.rs

```diff
@@ -292,22 +314,22 @@ pub mod test {
 
     impl ProverSM {
         fn to_presentation_prepared_state(mut self) -> ProverSM {
-            self = self.step(ProverMessages::PreparePresentation((_credentials(), _self_attested())), None).unwrap();
+            self = self.step(ProverMessages::PreparePresentation((_credentials(), _self_attested())), None::<&fn(&A2AMessage) -> _>).unwrap();
             self
         }
 
         fn to_presentation_sent_state(mut self) -> ProverSM {
-            let connection_handle = Some(mock_connection());
-            self = self.step(ProverMessages::PreparePresentation((_credentials(), _self_attested())), None).unwrap();
-            self = self.step(ProverMessages::SendPresentation, connection_handle).unwrap();
+            let send_message = Some(&|a2a_message: &A2AMessage| VcxResult::Ok(()));
```

Comment:
I would make it `Some(&|_: &A2AMessage| VcxResult::Ok(()))`, as the argument is not used